backend/controllers/contributions.js

```diff
@@ -0,0 +1,27 @@
+const axios = require("axios")
+const config = require("../config/index")
+
+const AuthContribution = {
+  async verifyIdentity(token) {
+    return await axios.get(
```

Comment:
no need for await here as we are returning the promise

Reply:
Thanks 👍